modules/tinymce/src/core/test/ts/browser/UndoManagerTest.ts

```diff
@@ -611,4 +612,80 @@ describe('browser.tinymce.core.UndoManagerTest', () => {
 
     assert.equal(changeEventCounter, 0, 'No events should be detected');
   });
+
+  context('dispatchChange', () => {
+    const initialContent = '<p>some inital content</p>';
+    const manualModifiedLevel = 'a modified last level';
+
+    const assertChangeEvent = (
+      event: { level: string; lastLevel: string },
+      expectedLevel: string | undefined,
+      expectedLastlevel: string | undefined
+    ) => {
+      assert.equal(event.level, expectedLevel, 'Level has not the expected content');
+      assert.equal(event.lastLevel, expectedLastlevel, 'Last level has not the expected content');
+    };
+
+    let changeEventCounter: number;
+    let currentChangeEvent: { level: string; lastLevel: string };
+
+    const onChange = (e: EditorEvent<{
+      level: UndoLevel;
+      lastLevel: UndoLevel;
```

Comment:
This type is also missing that `lastLevel` could be `undefined`.

Reply:
done